Item.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Item.h"
+
+Item::~Item() 
+{ 
+    
+}
+
+Item::Item(const std::string& name_, int effect) : name(name_), boost(effect)
+{
+
+}
+
+void Item::use(Character*)
+{
+
+}
```

Comment:
Don't implement a base class pure virtual function.
They're pure virtual to force derived classes to HAVE to implement them.
You take that requirement away when you implement them in the base class.
- [ ] revise